imagetest/junit.go

```diff
@@ -0,0 +1,98 @@
+package imagetest
+
+import (
+	"bytes"
+	"encoding/xml"
+	"fmt"
+
+	junitFormatter "github.com/jstemmer/go-junit-report/formatter"
+	junitParser "github.com/jstemmer/go-junit-report/parser"
+)
+
+type testSuites struct {
+	XMLName   xml.Name     `xml:"testsuites"`
+	Name      string       `xml:"name,attr"`
+	Errors    int          `xml:"errors,attr"`
+	Failures  int          `xml:"failures,attr"`
+	Tests     int          `xml:"tests,attr"`
```

Comment:
it would be fine to generate these values one-time, but they need to exist in the struct bc the struct represents the final junit output. the junit spec is pre-existing and we are conforming to it